doc/content/reference/tenant-management/_index.md

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Tenant Management'
+description: ''
+distributions: ['Enterprise']
+---
+
+This guide contains detailed information on how to perform common tenant management operations for {{% tts %}} Enterprise.
+
+<!--more-->
+
+{{< cli-only >}}
+
+## Authentication
+
+Instead of using API keys and OAuth access tokens as for other {{% tts %}} services, the tenant services require using **tenant admin keys** for authentication.
+
+> Tenant admin keys are configured in the Identity Server as described [here]({{< ref "/reference/configuration/identity-server#tenant-administration-options" >}}).
+
+This means all tenant management CLI commands will need to have the `--tenant-admin-key` flag appended, followed by a tenant admin key that you configured in the Identity Server. See the section below for examples.
+
+## Basic Operations
+
+### Create a Tenant
+
+To create a new tenant with the tenant ID `tenant1`:
+
+```
+$ tti-lw-cli tenant create tenant1 --tenant-admin-key 14596c0bdb9b11f6a97273e7c09167531571efa5898274e48660ee8d08f62b67
```

Comment:
Probably useful to declare a local variable;

```bash
$ tenantadminkey=14596c0bdb9b11f6a97273e7c09167531571efa5898274e48660ee8d08f62b67
$ tti-lw-cli tenant create tenant1 --tenant-admin-key $tenantadminkey
```

@adriansmares we don't respect an env key for this right?

Reply:
Correct, this is just an extra parameter added to the tenant commands - not a config parameter (when used like here).

Reply:
Indeed it doesn't respect the `TTN_LW_TENANT_ADMIN_KEY` env.

Another tip on macOS is to copy the tenant admin key (to the clipboard) and use `pbpaste`.
```
tenantadminkey=$(pbpaste)
$ tti-lw-cli tenant create tenant1 --tenant-admin-key $tenantadminkey
```